components/common/PrayerPoints.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Container, Card } from "react-bootstrap";
+import {FaPrayingHands} from 'react-icons/fa'
+import { IconContext } from "react-icons/lib";
+import { BsDownload } from "react-icons/bs";
+import Button from "react-bootstrap/Button";
+import { useTranslation } from "next-i18next";
+
+interface prayerProps {
```

Comment:
Maybe a question more for @jayjhan8? When do we use type vs interface? I think in my changes I generally use type...

Reply:
Yeah I'm not sure, interface is what we use at my workplace for typing props and that's just what I've been going with, so honestly im not sure but maybe Jay can speak to that?